### PR TITLE
Feat newlogin

### LINKED ISSUE #971: Welcome screen and login/create user components could be spruced up
## Abstract

The welcome and login screen is one of the first things users see when they open the application. Let's add some interesting elements and clean up the UI a bit to make rotki more inviting!

I have started doing some work on this and will share some examples so that we can discuss and vote in the coming days.

### PR DESCRIPTION
**Pull request by isidorosp**
Fix #971

* Added the flying rotki background to the Welcome Screen
* Cleaned up some of the component code between Sign In and Create New
Account screens
* Added transitions between the two screens
* Added "rotki" header to the Welcome Dialog
* De-emphasized "Create New Account" button in Sign In screen
* De-emphasized "Cancel" button in Create New Account screen
* Moved tooltip for "Enable Premium" from the checkbox to a new
(i) info icon

GIF below:
rotkiWelcomeNew

### CODE REVIEW

File: electron-app/src/components/AccountManagement.vue [hunk 1]
```diff
@@ -1,46 +1,73 @@
...
-      >
-        <div>Rotki is a local application that respects your privacy.</div>
-        <div>
-          Rotki accounts are encrypted using your password and saved in your
-          local filesystem.
+    <v-overlay class="account_management__loading">
+      <v-dialog :value="true && !premiumVisible" persistent max-width="450">
+        <v-card class="rotki-welcome pb-6">
+          <div
+            class="pt-6 pb-3 display-3 font-weight-black primary white--text"
```
Comment (kelsos) [inline electron-app/src/components/AccountManagement.vue:7]: Interesting, I had absolutely no idea about these helper classes in vuetify

File: electron-app/src/components/AccountManagement.vue [hunk 2]
```diff
@@ -164,6 +191,32 @@ export default class AccountManagement extends Vue {
...
+@keyframes scrollLarge {
+  0% {
+    transform: rotate(-13deg) translateY(0);
+  }
+  100% {
+    transform: rotate(-13deg) translateY(-1200px);
+  }
+}
+
+.account_management__loading {
```
Comment (kelsos) [inline electron-app/src/components/AccountManagement.vue:203]: nitpicky: but the rul for loading is already at 229 &__loading nested in .account_management in scss translates to .account_management__loading

File: electron-app/src/components/AccountManagement.vue (hunk 1 above)
Comment (isidorosp) [reply electron-app/src/components/AccountManagement.vue:7]: Yes I read about them by chance when I was reading about Tailwind. They are great!

File: docs/changelog.rst [hunk 3]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`971` Rotki's initial loading and welcome screens are now integrated with an improved UI and a scrolling rotkehlchen in the background to welcome the user.
```
Comment (LefterisJP) [inline docs/changelog.rst:5]: Perhaps swap rotkehlchen with robin here.

**Comment by LefterisJP** [comment]
The travis failure is due to OSX travis never starting ... ignored it and merged

**Pull request merged**